common/depot/build_hashed_repo_name.go

```diff
@@ -29,7 +24,7 @@ func BuildHashedRepoName(author string, repo string, sha string) string {
 		author,
 		len(repo),
 		repo,
-		sha[:6])
```

Comment:
Is there any particular reason you want to use, what I assume to be, the entire 40 character SHA here?